cosmoz-data-nav.js

```diff
@@ -515,13 +533,13 @@
 				return;
 			}
 			const path = Polymer.dom(event).path,
-				attr = this.selectAttribute;
+				attr = this.selectAttribute,
+				selectEl = path.find(e => e && e.hasAttribute && e.hasAttribute(attr));
 
-			let select = path.find(e => e && e.hasAttribute && e.hasAttribute(attr));
-			if (!select || select.closest(this.is) !== this) {
+			if (!selectEl || path.slice(path.indexOf(selectEl)).find(e => e.is && e.is === this.is) !== this) {
```

Comment:
Please extract `path.slice(path.indexOf(selectEl))` and the .find() to two different const's to describe them

Reply:
It should even be possible to replace the `find` + `indexOf` + `find` (again) combination with a single... `for` loop.
Just kidding 😂.